### DISCUSSION TITLE
[Question][Hw2] ntp server not responding at all

### DISCUSSION THREAD

**Discussion by jirheee** [prosyslab-classroom/cs348-information-security#507] [Q&A]
I tried many experiments on attack using dns, so I started to use ntp service. 

But whatever I do, the ntp server doesn't respond.

I tried to communicate with ntp server by following methods, but no respond is returned

(1) using scapy to send NTP packet, (both plain NTP Header and mode 7) to 192.168.20.1
(2) use ntpdc -> it says Name or service not known,,
(3) use Nmap to scan if the amp server has the functionality I want

But by whatever method, nothing returns back to my attacker,, What am I doing wrong? Can I ask for some starting point?

**Comment by jirheee** [comment]
When I look into the record database, it looks like this.

`{ "_id" : ObjectId(), "proto" : "UDP", "port" : 123, "timestamp" : , "reqbytes" : 669240, "resbytes" : 0, "hmac" : }`

**Comment by miraliahmadli** [comment]
I am having the same issue. Please let me know if you find any solution.

**Comment by ramel0915** [comment]
Some NTP servers disabled mode 7 due to security issues. Use another mode.
